developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst

```diff
@@ -38,6 +38,34 @@ Nextcloud now provides meaningful default styles for heading elements.
 This can cause visual regressions if your code does not explicitly set font size and weight.
 If you need to use heading elements outside of text content, you might need to adjust their styles.
 
+Line height
+^^^^^^^^^^
+
+`--default-line-height` CSS variable is a unitless value `1.5` instead of a unit value `24px`:
+- `line-height` of an element with a non-default font size is now adjusted to the element's font size
+- Unit calculation isn't valid anymore. Use [lh](https://developer.mozilla.org/en-US/docs/Web/CSS/length#lh) unit instead.
+
+```diff
```

Comment:
the docs are not markdown :speak_no_evil: 
it's rst: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/code_blocks.html